index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Template 3</title>
+</head>
+<body>
+    <header class="topbar">
+        <h3>Welcome</h3>
```

Comment:
- [ ] Kindly use an `a` tag here instead of `h3`. This can serve as a link to the home page later.